src/lib/RegistrationSchema.svelte

```diff
@@ -4,17 +4,29 @@
     import { TextInputDto } from './model/text-input-dto';
     import { invoke } from '@tauri-apps/api/tauri';
     import { settings } from './util/settings';
+    import { type Writable } from 'svelte/store';
     import { v4 } from 'uuid';
     import { path } from '@tauri-apps/api';
-    import { onMount } from 'svelte';
+    import { onDestroy, onMount } from 'svelte';
     import { isLoggedIn } from './Auth.svelte';
+    import { appWindow } from '@tauri-apps/api/window';
+    import type { Event, UnlistenFn } from '@tauri-apps/api/event';
+    import { type AllTransferProgress, calculateProgress, type TransferProgress } from './model/transfer-progress';
```

Comment:
```suggestion
    import { type AllTransferProgress, type TransferProgress } from './model/transfer-progress';
```
Fjerne ubrukt import